### ISSUE TITLE
Design GUI for steering wheel area on starting page

### ISSUE THREAD

**Issue by stefanieschreiner** [amosproj/amos2022ws02-automotive-test-app#19]
## User story
1.	As a User
2.	I want to be able to enter an area in the test app where all use cases for simulating steering wheel buttons are collected 
3.	So that I can easily switch between the use cases contexts.

With this User story the graphical user interface of the steering wheel button area on the starting page of the app is to be designed.

## Acceptance criteria
* The graphical user interface of the steering wheel area on the starting page of the app is designed.
* An XML for the GUI of the steering wheel area on the starting page of the app has been created in Android studio.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
